src/viewport/index.js

```diff
@@ -1,2 +1,6 @@
-export {default as default} from './viewport';
-export {default as Viewport} from './viewport';
+export * from './viewport';
+export {default as BaseViewport} from './viewport';
+export * from './webgl-viewport';
+export {default as WebGLViewport} from './webgl-viewport';
+export {default as Viewport} from './webgl-viewport';
```

Comment:
could you add a comment that Viewport is for legacy support (and will be removed in future releases)?